.taprc

```diff
@@ -0,0 +1,7 @@
+timeout: 50
+jobs: 1
+statements: 95
+branches: 90
+functions: 95
+lines: 98
+test-env: NODE_OPTIONS=--no-warnings
```

Comment:
Once https://github.com/clinicjs/node-clinic-bubbleprof/issues/383 is not solved yet, the warning messages displayed by nodejs affect the async_hooks events.